src/Utils/SchemaExtender.php

```diff
@@ -296,15 +294,22 @@ protected static function extendImplementedInterfaces(ImplementingType $type): a
             /** @var ObjectTypeExtensionNode|InterfaceTypeExtensionNode $extension */
             foreach ($extensions as $extension) {
                 foreach ($extension->interfaces as $namedType) {
-                    $interfaces[] = static::$astBuilder->buildType($namedType);
+                    /** @var InterfaceType $interface */
+                    $interface    = static::$astBuilder->buildType($namedType);
+                    $interfaces[] = $interface;
                 }
             }
         }
 
         return $interfaces;
     }
 
-    protected static function extendType($typeDef)
+    /**
+     * @param ListOfType|NonNull|(Type &NamedType) $typeDef
```

Comment:
```suggestion
     * @param ListOfType|NonNull|(Type&NamedType) $typeDef
```

Reply:
I would prefer that, PHP_CS keeps messing it up. I don't like that tool, php-cs-fixer does a much better job in my experience.